go/vt/vtgate/planbuilder/operators/operator.go

```diff
@@ -90,10 +89,10 @@ func (noInputs) Inputs() []ops.Operator {
 
 // AddColumn implements the Operator interface
 func (noColumns) AddColumn(*plancontext.PlanningContext, sqlparser.Expr) (int, error) {
-	return 0, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "this operator cannot accept columns")
+	return 0, vterrors.VT13001("this operator cannot accept columns")
 }
 
 // AddPredicate implements the Operator interface
 func (noPredicates) AddPredicate(*plancontext.PlanningContext, sqlparser.Expr) (ops.Operator, error) {
-	return nil, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "this operator cannot accept predicates")
+	return nil, vterrors.VT13001("this operator cannot accept columns")
```

Comment:
hmm.. that should be `predicates` still?

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)